app/assets/javascripts/bastion/components/views/bst-bookmark.html

```diff
@@ -0,0 +1,51 @@
+<li ng-show="bookmarks.length === 0" translate>
+  No Bookmarks
+</li>
+<li ng-repeat="bookmark in bookmarks">
+  <a ng-click="setQuery(bookmark)" class="ng-scope">{{ bookmark.name }}</a>
+</li>
+<li class="divider"></li>
+<li>
+  <a ng-click="add()" translate class="ng-scope">Bookmark this search</a>
+</li>
+<li>
+  <a ng-href="http://www.theforeman.org/manuals/1.10/index.html#4.1.5Searching" data-id="aid_manuals_1.10_index.html" rel="external" target="_blank"><i class="glyphicon glyphicon-question-sign icon-black"></i>
+    <span translate> Documentation</span>
+  </a>
+</li>
```

Comment:
I suppose we can keep this for parity, but it's kind of silly due to needing updates every time there is a release.